### ISSUE TITLE
[Code lens] No code lens for constructors

### ISSUE THREAD

**Issue by fuszenecker** [ionide/ionide-vscode-fsharp#250]
I observed that there is no code lens support for constructors, i.e.

```
   type Vector2D =
        struct
            val X: float
            val Y: float

            new(x, y) = { X = x; Y = y }
        end
```

The phenomenon is the same for classes, too.

**Comment by fuszenecker** [comment]
Btw, Ionide-FSharp is the best plug-in in the world! :-)

**Comment by vasily-kirichenko** [comment]
Fixed in https://github.com/ionide/FsAutoComplete/pull/25

![image](https://cloud.githubusercontent.com/assets/873919/19805273/0f567e70-9d1c-11e6-803f-d497d82d3ba0.png)

**Comment by Krzysztof-Cieslak** [comment]
Released in `2.8.7`

**Comment by fuszenecker** [comment]
Thank you very much!
